src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2274,6 +2274,26 @@ public bool EnableControlStatePropagation
             }
         }
 
+        /// <summary>
+        /// Whether the actor grab all touches even if touch leaves its boundary.
+        /// </summary>
+        /// <returns>true, if it grab all touch after start</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool GrabTouchAfterLeave
+        {
+            get
+            {
+                bool temp = false;
+                GetProperty(View.Property.CAPTURE_ALL_TOUCH_AFTER_START).Get(out temp);
```

Comment:
I think you have decided to change from capture to grab. 
so could you change CAPTURE_ALL... name as Grab... ?
also I think capital letter use is not preferred. could you change all the member variables or properties names as Pascal case? ^^;

https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/